web/blog/tenzir-v4.9/index.md

```diff
@@ -46,10 +112,31 @@ export --live
 | enrich indicators --field dns.rrname
 ```
 
+The `enrich` operator gained a new `--filter` option to remove events it could
+not enrich. Use the new option to remove anything that is not included in the
+Bloom filter:
+
+```
+export --live
+| where #schema == "suricata.dns"
+| enrich indicators --field dns.rrname --filter
+```
+
 ## Housekeeping
 
+Other noteworthy changes and improvements:
+- `tenzir.db-directory` is now `tenzir.state-directory`. The old option remains
+  functional, but will be phased out in an upcoming release.
+- On the command-line, Tenzir now respects [`NO_COLOR`](https://no-color.org)
+  when printing diagnostics.
```

Comment:
And we do not print colors if we output device is not a terminal, making it suitable for non-interactive use, which is IMHO the more important part of the change. But you can leave as-is, I don't really mind.

Reply:
Added a sentence.